deploy/helm/values.yaml

```
@@ -271,3 +271,38 @@ autoupdate:
   ## @param autoupdate.scheduler - Schedule cron job to check & update Helm image
   ##
   scheduler: "0 * * * *"
+
+applicationConfig:
+  APPSMITH_SENTRY_DSN: ""
+  APPSMITH_SMART_LOOK_ID: ""
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_ID: ""
+  APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET: ""
+  APPSMITH_SEGMENT_KEY: ""
+  APPSMITH_RAPID_API_KEY_VALUE: ""
+  APPSMITH_MARKETPLACE_ENABLED: ""
+  APPSMITH_OPTIMIZELY_KEY: ""
+  APPSMITH_ALGOLIA_API_ID: ""
+  APPSMITH_ALGOLIA_API_KEY: ""
+  APPSMITH_ALGOLIA_SEARCH_INDEX_NAME: ""
```

Let's remove internal variables here as well.

```suggestion
  APPSMITH_OAUTH2_GOOGLE_CLIENT_ID: ""
  APPSMITH_OAUTH2_GOOGLE_CLIENT_SECRET: ""
  APPSMITH_OAUTH2_GITHUB_CLIENT_ID: ""
  APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET: ""
```